docs/getting-started/olmv1_getting_started.md

```diff
@@ -2,9 +2,9 @@
 
 The following script will install OLMv1 on a Kubernetes cluster. If you don't have one, you can deploy a Kubernetes cluster with [KIND](https://sigs.k8s.io/kind).
 
-> [!CAUTION]  
-> Operator-Controller depends on [cert-manager](https://cert-manager.io/). Running the following command
-> may affect an existing installation of cert-manager and cause cluster instability.
+!!! warning
+    Operator-Controller depends on [cert-manager](https://cert-manager.io/). Running the following command
+    may affect an existing installation of cert-manager and cause cluster instability.
```

Comment:
`!!! warning` This renders well in our mkdocs website, but not on GitHub. `[!CAUTION]` renders well on GitHub, but not in mkdocs.

Ref: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#classic-admonitions-docsstylesheetsextracss

Reply:
Moved to https://github.com/operator-framework/operator-controller/pull/1446

Resolving